Makefile

```diff
@@ -53,7 +53,9 @@ tools: pd-tso-bench pd-recover pd-analysis pd-heartbeat-bench
 pd-server: export GO111MODULE=on
 ifeq ("$(WITH_RACE)", "1")
 pd-server:
+ifneq ($(OS),Windows_NT)
```

Comment:
why can not build it on windows?

Reply:
It's just because the sh script may not be executed successfully in Windows. If necessary later, we may need to implement the powershell version of this script.